src/smufl_bravura_larger_noteheads.lua

```diff
@@ -0,0 +1,46 @@
+function plugindef()
+    -- This function and the 'finaleplugin' namespace
+    -- are both reserved for the plug-in definition.
+    finaleplugin.Author = "Jacob Winkler"
+    finaleplugin.Copyright = "2022"
+    finaleplugin.Version = "1.0"
+    finaleplugin.Date = "4/18/2022"
+    finaleplugin.Notes = [[
+Replaces the noteheads in a 'Bravura' document with the larger glyphs included in the Stylistic Alternates category. These alternate glyphs (which are about 8% larger than the ones loaded by default) are the ones that the font is actually designed to use, and are the ones used by Dorico. For a discussion about these larger noteheads and why they are not used by default, see this thread in the MuseScore forum: https://musescore.org/en/node/68461
+    ]]
+    return "Bravura: Large Noteheads", "Bravura: Large Noteheads", "Replaces noteheads in Bravura with the intended larger size."
+end
+
+--JETSTREAM-- should add to library!

```

Comment:
What is this comment for?

If you think it should be added to the library, you can just add it to the library.